packages/fast-components-react-msft/app/utilities/additional-props.plugin.ts

```diff
@@ -13,10 +13,14 @@ export default class MapChildrenPropToCallbackPassingClassName extends Plugin<
             const callbackArguments: any = {
                 className,
             };
-            callbackArguments.disabled =
-                typeof disabled === "boolean" ? disabled : undefined;
-            callbackArguments.appearance =
-                typeof appearance === "string" ? appearance : undefined;
+
+            if (callbackArguments.disabled) {
```

Comment:
Is this really what we want? These conditionals will never evaluate to true because `callbackArguments` is created directly above and never assigned either a `disabled` or `appearance` property - meaning they will always be `undefined`

Reply:
Nope! Sorry, that was a logic error on my part. I've updated.